src/Integration/UseShopwareExceptionIgnores.php

```
@@ -5,34 +5,27 @@
-
-            $config = $exceptions[$eventExceptions->getType()] ?? [];
-
-            if (!isset($config['log_level'])) {
-                return $event;
-            }
-
-            if ($config['log_level'] === 'notice') {
-                return null;
-            }
+            $exceptions = $eventDispatcher->dispatch(new FilterExceptionEvent($exceptions))->getExceptions();
+            $event->setExceptions($exceptions);
```

PHPStan (2.1.15)

Parameter #1 $exceptions of method Sentry\Event::setExceptions() expects array<Sentry\ExceptionDataBag>, array given.


(argument.type)